controller_manager_tests/CMakeLists.txt

```diff
@@ -28,30 +42,45 @@ target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
 add_executable(dummy_app src/dummy_app.cpp)
 target_link_libraries(dummy_app ${PROJECT_NAME} ${catkin_LIBRARIES})
 
+
+#############
+## Testing ##
+#############
+
 if(CATKIN_ENABLE_TESTING)
   find_package(rostest REQUIRED)
+
   add_executable(cm_test test/cm_test.cpp)
   add_dependencies(tests cm_test)
-  target_link_libraries(cm_test ${GTEST_LIBRARIES} ${catkin_LIBRARIES})
+  target_link_libraries(cm_test ${catkin_LIBRARIES} ${GTEST_LIBRARIES})
+
   add_rostest(test/cm_test.test)
+
   catkin_add_nosetests(test)
+
   add_rostest(test/cm_msgs_utils_rostest.test)
+
   add_rostest(test/controller_manager_scripts.test)
 endif()
 
-# Install
+
+#############
+## Install ##
+#############
+
+# Install headers
 install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+)
 
-# Install library
-install(TARGETS ${PROJECT_NAME}
+# Install targets
+install(TARGETS ${PROJECT_NAME} dummy_app
```

Comment:
again, not sure it's good to install this

Reply:
Fixed in 6f4648d